### ISSUE TITLE
Project management by github

### ISSUE THREAD

**Issue by SomeoneElseOSM** [osmfoundation/ewg_bidding#3]
You appear to be trying to manage a project via github.  Please don't.  How on earth are normal humans supposed to understand what is implied by https://github.com/osmfoundation/ewg_bidding/pull/2 ?  Nothing in the readme at https://github.com/osmfoundation/ewg_bidding/blob/main/README.md says what the EWG is or how it came to be, or what you're trying to achieve.  Even https://github.com/osmfoundation/ewg_bidding/blob/draft/projects/ability-to-block-other-users/README.md (once updated with input from https://community.openstreetmap.org/t/rfc-on-engineering-working-group-project/1542 and https://github.com/openstreetmap/openstreetmap-website/issues/3129 ) is extremely light on detail.

With regard to your first "project", with a DWG hat on, after reading https://github.com/osmfoundation/ewg_bidding/blob/draft/projects/ability-to-block-other-users/README.md I simply have no idea what the implications will be.  If the net effect is to allow mappers to cut themselves off from the community and willfully ignore helpful advice from other mappers until they are hard-blocked by the DWG, then that will be a bad outcome.  Unfortunately https://github.com/osmfoundation/ewg_bidding/blob/draft/projects/ability-to-block-other-users/README.md is even unclear what "messages" means.

**Comment by drolbr** [comment]
We are happy about all feedback.

> You appear to be trying to manage a project via github.

Github has been chosen, because we do not want to lose bidder because of annoying tools. The wiki has its own syntax, and far worse, its own social conventions. Discourse has no serious change tracking, and mailing lists have in these days mixed reception, too. Most likely most or all projects will have a pull request as a deliverable, so using Github for communications means zero tool overhead for bidders.

We have learned from the rather terse description that there is room for misinterpretation from multiple completely unexpected directions: the status of email notifications, confusion about "blocking" in DWG context versus the actual "muting" here, expectations of a social feature and fear it would relate to changeset comments.

Similarly, your feedback measn that we will include more links to the OSMF wiki EWG pages such that it is easier for people to find information about the EWG.

> If the net effect is to allow mappers to cut themselves off from the community.

Private messages from user to user. I understand that based on your question one might subsume changeset comments there, too, but those are out of scope.

**Comment by mmd-osm** [comment]
Can I suggest that EWG member @rkoeze who seems to be main person working on drafting proposal document joins the discussion on the community site over at https://community.openstreetmap.org/t/rfc-on-engineering-working-group-project/1542

I feel the communication is still a bit too indirect. You're of course free to organize your WG internal communication is whatever way works best for you. However, to be quite honest, as an outside observer who's not involved at all in this process, and don't intends to be, it's somehow difficult to find out why much of the original feedback, e.g. by @simonpoole and others hasn't been considered along the process.

**Comment by rkoeze** [comment]
Most definitely. And thank you for the feedback, both here and on the community site.

One thing we've learned from the process so far is that we need to work with stakeholders (project maintainers, working groups, etc.) earlier in the process. We'll be documenting that in [the project framework](https://wiki.osmfoundation.org/wiki/Engineering_Working_Group/Project_Funding_Framework) and making sure it's part of our process going forward. I think, perhaps naively, we'd believed the public review phase would be the place to do that. That was incorrect.

We'll be incorporating the feedback into the project description. I will share an updated version in the community site, as well as any points requiring further clarity or consensus.

**Comment by mmd-osm** [comment]
@rkoeze : since Roland has asked for another round of feedback, I wanted to double check with you, if you still had plans for an updated version, or if I maybe missed this somehow?

**Comment by rkoeze** [comment]
Thanks for catching that @mmd-osm. I've made a PR for those updates [here](https://github.com/osmfoundation/ewg_bidding/pull/4). I will send a follow up to Roland's email with the link as well.

**Comment by simonpoole** [comment]
I believe there is still an inconsistency in the spec: as it exempts changeset discussions, but these are, at least currently, not notified via the internal messaging system to start with (which is a problem, but a different one that that at hand).
